flopy/discretization/structuredgrid.py

```diff
@@ -239,21 +241,32 @@ def intersect(self, x, y, local=False):
 
         """
         # transform x and y to local coordinates
-        x, y = super(StructuredGrid, self).intersect(x, y, local)
+        x, y = super(StructuredGrid, self).intersect(x, y, local, forgive)
 
         # get the cell edges in local coordinates
         xe, ye = self.xyedges
 
         xcomp = x > xe
         if np.all(xcomp) or not np.any(xcomp):
-            raise Exception('x, y point given is outside of the model area')
-        col = np.where(xcomp)[0][-1]
+            if forgive:
+                col = np.nan
+            else:
+                raise Exception(
+                    'x, y point given is outside of the model area')
+        else:
+            col = np.where(xcomp)[0][-1]
 
         ycomp = y < ye
         if np.all(ycomp) or not np.any(ycomp):
-            raise Exception('x, y point given is outside of the model area')
-        row = np.where(ycomp)[0][-1]
-
+            if forgive:
+                row = np.nan
+            else:
+                raise Exception(
+                    'x, y point given is outside of the model area')
+        else:
+            row = np.where(ycomp)[0][-1]
+        if np.any(np.isnan([row, col])):

```

Comment:
Suggest returning `None` for whichever failed. For example, `None, 16`, `10, None`, `None, None`.

Reply:
Hi @jdhughes-usgs, can change to return `None` but `np.nan` is generally easier to work with downstream with both numpy and pandas (see @jwhite-usgs comment below) . Re. returning `NaN` (or `None`) for both, I guess was figuring that if is it outside grid for one it is outside the grid for both row and col and therefore shouldn't be assigned a value for either, happy to change to just return `NaN` (or `None`) for whichever is out of grid `x` or `y` range, if there is a strong preference for that.